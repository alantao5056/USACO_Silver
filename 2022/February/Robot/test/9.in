40
14125 16811
517 1939
3178 2863
1887 4937
-7403 -5667
-1151 -1565
8232 9181
1426 444
6975 7026
-2908 -6238
-7704 -8609
7582 6584
-1570 -1684
7536 4733
3363 3942
-573 22
3253 2302
22 -2575
5216 8078
1650 4058
3308 3099
-5174 -2550
7472 7282
5273 3011
-5441 -3980
747 1180
9884 7208
-939 25
-9047 -9091
5345 2197
-5230 -7666
2802 3207
2643 89
6351 5333
-4511 -4575
8468 7466
-9255 -6556
-5773 -8417
-3672 -6004
-2912 397
7150 7764
