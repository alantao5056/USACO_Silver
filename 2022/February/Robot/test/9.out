0
0
0
0
0
0
0
0
0
0
1
6
16
32
46
61
94
124
82
118
114
87
70
38
33
12
8
5
1
1
0
0
0
0
0
0
0
0
0
0
